common/source/docs/common-airspeed-sensor.rst

```diff
@@ -0,0 +1,31 @@
+.. _common-airspeed-sensor:
+
+[copywiki destination="copter,rover"]
+
+Airspeed Sensor
+===============
+
+Copter, Rover, and Blimp can attach and enable an Airspeed Sensor. However, unlike Plane which uses the readings for navigation and control, these vehicles only log and report the readings for use in displays in Ground Control Stations, OSDs, and telemetry. The Plane airspeed sensor :ref:`documentation<plane:airspeed>` can followed for selecting and mounting the type of airspeed sensor, but be aware that it will not provide the functionality beyond measuring and reporting discussed in Plane.
```

Comment:
```suggestion
Copter, Rover, Sub, and Blimp can attach and enable an Airspeed Sensor. However, unlike Plane which uses the readings for navigation and control, these vehicles only log and report the readings for use in displays in Ground Control Stations, OSDs, and telemetry. The Plane airspeed sensor :ref:`documentation<plane:airspeed>` can followed for selecting and mounting the type of airspeed sensor, but be aware that it will not provide the functionality beyond measuring and reporting discussed in Plane.
```

Reply:
Actually Sub is an independently maintained Wiki...you wont find SUB stuff in the other vehicle's wiki's